arrow/src/test/java/org/apache/calcite/adapter/arrow/ArrowTest.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.calcite.adapter.arrow;
+
+import org.apache.calcite.jdbc.CalciteConnection;
+import org.apache.calcite.schema.Table;
+import org.apache.calcite.util.Source;
+
+import org.apache.calcite.util.Sources;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.util.Map;
+
+public class ArrowTest {
+
+  @Test void testArrowSchema() {
+    Source source = Sources.of(ArrowTest.class.getResource("/files"));
+    ArrowSchema arrowSchema = new ArrowSchema(source.file().getAbsoluteFile());
+    Map<String, Table> tableMap = arrowSchema.getTableMap();
```

Comment:
@michaelmior I have one doubt here. `getRowType` method in `ArrowTable` class takes in an argument of type `RelDataTypeFactory`. I want to call `getRowType` method for each table in the map. So while calling this method what argument should I pass in? I tried understanding it but I couldn't really figure out what I should be passing in to that method.

Reply:
You should just be able to use new `new JavaTypeFactoryImpl(RelDataTypeSystem.DEFAULT)` for testing purposes